README.md

```diff
@@ -17,138 +19,143 @@ Before getting started with this project, please get yourself familiar with the
     ```shell
-    git clone --branch $adu_release http://github.com/azure/meta-raspberrypi-adu
+    git clone --branch $adu_release http://github.com/azure/meta-iot-hub-device-update-delta
     ```
 
-### How To Build The Project Locally
+Next we move on to how to build the project assuming you've setup the project like above instructions. If you don't follow the setup instructions you will have to make modifications to `scripts/build.sh` to make sure the build works. 
+
+### Building The Project Locally
 
 #### Install Build Dependencies and Tools
 
-For more information, see [Yocto Project Quick Build](https://docs.yoctoproject.org/brief-yoctoprojectqs/index.html#yocto-project-quick-build)
+For more information on the Yocto build system, the open embedded base image, and example builds please see [Yocto Project Quick Build](https://docs.yoctoproject.org/brief-yoctoprojectqs/index.html#yocto-project-quick-build). 
 
-```sh
-# Install build dependencies
-sudo ./scripts/install-deps.h
+Please look into `scripts/install-deps.sh` to determine what you may need to integrate into your build as you move forward with the project. 
 
-# Checkout a desired 'poky' branch
-cd yocto/poky
-git fetch
-git checkout -t origin/honister -b my-honister
-git pull
 
-# Initialize build environment
-source oe-init-build-env
+1. From the `project_root` please execute the following command in your terminal
+```shell
+sudo ./scripts/install-deps.h
 ```
 
+### Creating the Private Key for Sw Update Signing
+
+To create the `*.swu` file you will need to provide the build system with a private key and password file so that it can sign the generated image and then create the Sw Update file. This is REQUIRED for a Sw Update update to function. You MUST put the private key and password file inside of the `repo-root-directory/keys` directory. The build will break if you do not complete this step and your GitHub bug will elicit an exasperated sign from the developers of Device Update. Be warned. 
+
+You can find the instructions for generating the private key and creating the password file [here](./keys/README.md). 
+
+
 ### Build The Project
 
-```sh
-# Run from project root folder
-./scripts/build.sh -c -t $BUILD_TYPE -v $BUILD_NUMBER -o $BUILD_OUTPUT_DIR [optional build arguments]
-         
+To build the project you can either use our helper script or read the `build.sh` script and use your own terminal  commands to build the layer. Keep in mind Yocto builds can take time depending on your machine. It's best to use a local cache if you're going to be running multiple builds. We use the `-o` option to specify the output directory which in turn builds a local cache that can expediate your local builds. An example invokation is specified below. It is executed from the repositories root folder. NOT the `yocto` directory.
```

Comment:
Typo: _An example **invokation** is specified below_

Reply:
Weird random fun fact: expediate is not actually a word but comes from an error in copying a dictionary so technically that is also a typo.